test/pod.bats

```diff
@@ -13,82 +13,36 @@ function teardown() {
 # PR#59
 @test "pod release name on remove" {
 	start_crio
-	run crictl runp "$TESTDATA"/sandbox_config.json
-	echo "$output"
-	[ "$status" -eq 0 ]
-	id="$output"
-	run crictl stopp "$id"
-	echo "$output"
-	[ "$status" -eq 0 ]
-	run crictl rmp "$id"
-	echo "$output"
-	[ "$status" -eq 0 ]
-	run crictl runp "$TESTDATA"/sandbox_config.json
-	echo "$output"
-	[ "$status" -eq 0 ]
-	id="$output"
-	run crictl stopp "$id"
-	echo "$output"
-	[ "$status" -eq 0 ]
-	run crictl rmp "$id"
-	echo "$output"
-	[ "$status" -eq 0 ]
+	id=$(crictl runp "$TESTDATA"/sandbox_config.json)
+	crictl stopp "$id"
```

Comment:
Cleanup is not needed as it's done in teardown. Should be removed in a separate patch, similar to e.g. https://github.com/cri-o/cri-o/pull/4253/commits/601e1e4cac7b930c1f1e42f8720d8cbdf9014c52

Reply:
Unless, of course, it's part of the test, like in "pod remove".

Reply:
I think it is part of the test, since the test is supposed to check if the pod name is released when it's removed?

Reply:
think it is part of the test, since the test is supposed to check if the pod name is released when it's removed

Reply:
I meant the second cleanup at the end of the function (lines 20-21 now).

Reply:
oh, I see what you mean